### PR TITLE
O365 org setting workaround

### PR DESCRIPTION
**Pull request by NikCharlebois**
#### Pull Request (PR) description
* O365OrgSettings
  * Introduced a workaround to fix an issue with the ExchangeOnlineManagement module where if connected to Security and Compliance center
    an error about an invalid token would get thrown when calling the Get-DefaultTenantMyAnalyticsFeatureConfig cmdlet.

#### This Pull Request (PR) fixes the following issues
* N/A

### CODE REVIEW

File: Modules/Microsoft365DSC/DSCResources/MSFT_O365OrgSettings/MSFT_O365OrgSettings.psm1 [hunk 1]
```diff
@@ -156,6 +156,14 @@ function Get-TargetResource
     $ConnectionModeTasks = New-M365DSCConnection -Workload 'Tasks' `
         -InboundParameters $PSBoundParameters
 
+    # Workaround for issue when if connected to S+C prior to calling cmdlet, an error about an invalid token is thrown.
```
Comment (andikrueger) [inline Modules/Microsoft365DSC/DSCResources/MSFT_O365OrgSettings/MSFT_O365OrgSettings.psm1:159]: Shouldn't we add this within MSCloudLoginAssistant or is this just the case for this resource?

Comment (NikCharlebois) [reply Modules/Microsoft365DSC/DSCResources/MSFT_O365OrgSettings/MSFT_O365OrgSettings.psm1:159]: Just this resource.

Comment (NikCharlebois) [reply Modules/Microsoft365DSC/DSCResources/MSFT_O365OrgSettings/MSFT_O365OrgSettings.psm1:159]: A bug was raised with the EXO Management Shell team about it.

Comment (andikrueger) [reply Modules/Microsoft365DSC/DSCResources/MSFT_O365OrgSettings/MSFT_O365OrgSettings.psm1:159]: ok, then!

**Pull request merged**